src/codegen/sdk/core/import_resolution.py

```diff
@@ -607,6 +607,8 @@ def names(self) -> Generator[tuple[str, Self | WildcardImport[Self]], None, None
                     imp.file.invalidate()
 
                 return
+            elif self.resolved_symbol is None:
```

Comment:
Can we break this part out into it's own pr with a test case. This specific change looks good